wqflask/wqflask/correlation_matrix/show_corr_matrix.py

```diff
@@ -165,16 +167,13 @@ def __init__(self, start_vars):
 
         self.pca_works = "False"
         try:
-            corr_result_eigen = np.linalg.eig(np.array(self.pca_corr_results))
-            corr_eigen_value, corr_eigen_vectors = sortEigenVectors(
-                corr_result_eigen)
 
             if self.do_PCA == True:
```

Comment:
**nipick**: I know you haven't touched this, but what does this return? If it's just a boolean, how about:
```suggestion
            if self.do_PCA:
```

Reply:
@BonfaceKilz   thanks for pointing this out.am also refactoring the whole module on a different pr